apex/contrib/csrc/gpu_direct_storage/gds.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+
+#ifndef __TORCH_GDS_H__
+#define __TORCH_GDS_H__
```

Comment:
would there happen to be some reasons to use header guards instead?
```suggestion
#pragma once
```